e2e/src/aws_kms.rs

```diff
@@ -0,0 +1,149 @@
+use anyhow::Context;
+use aws_sdk_kms::config::{Credentials, Region};
```

Comment:
You can import them from `fuels::*` better to try that way first since we're now committed to reexporting their types as part of our public API.